src/cli/commands/import.js

```diff
@@ -254,7 +254,10 @@ class ImportPackageResolver extends PackageResolver {
     }
   }
 
-  async init(deps: DependencyRequestPatterns, isFlat: boolean): Promise<void> {
+  async init(
+    deps: DependencyRequestPatterns,
+    {isFlat, isFrozen}: {isFlat: boolean, isFrozen?: boolean},
```

Comment:
Nice! I thought about default params but wasn't sure if we supported them. Since now I know, I'll go ahead and use it.